### PR TITLE
EVAKA-4401 absence type info

### PR DESCRIPTION
**Pull request by jaahon**
#### Summary

- Remove obsolete `TEMPORARY_VISITOR` absence type stuff
- Update Espoo absence type labels
- Add absence type info to legend

### CODE REVIEW

File: frontend/src/lib-customizations/espoo/citizen/assets/i18n/fi.tsx [hunk 1]
```diff
@@ -245,7 +245,7 @@ export default {
       absenceType: 'Poissaolon syy',
       absenceTypes: {
         SICKLEAVE: 'Sairaus',
-        OTHER_ABSENCE: 'Muu poissaolo',
+        OTHER_ABSENCE: 'Poissaolo',
         PLANNED_ABSENCE: 'Säännöllinen vapaapäivä/vuorohoito'
```
Comment (kimifoud) [inline frontend/src/lib-customizations/espoo/citizen/assets/i18n/fi.tsx:249]: sv and en need to be updated too

File: frontend/src/lib-customizations/common.tsx [hunk 2]
```diff
@@ -62,7 +62,6 @@ export const absenceColors = {
   FORCE_MAJEURE: status.danger,
   FREE_ABSENCE: accents.a7mint,
   TEMPORARY_RELOCATION: status.warning,
-  TEMPORARY_VISITOR: status.warning,
```
Comment (kimifoud) [inline frontend/src/lib-customizations/common.tsx:65]: One instance is still present here https://github.com/espoon-voltti/evaka/blob/3b294a9a85664df5f514cf53e410687c5e6be6a9/frontend/src/lib-customizations/espoo/employee/assets/i18n/fi.ts#L2160

File: frontend/src/lib-customizations/espoo/employee-mobile-frontend/assets/i18n/fi.ts [hunk 3]
```diff
@@ -90,12 +90,11 @@ export const fi = {
   absences: {
     title: 'Poissaolomerkintä',
     absenceTypes: {
-      OTHER_ABSENCE: 'Muu poissaolo',
+      OTHER_ABSENCE: 'Poissaolo',
       SICKLEAVE: 'Sairaus',
       UNKNOWN_ABSENCE: 'Ilmoittamaton poissaolo',
-      PLANNED_ABSENCE: 'Suunniteltu poissaolo / vuorohoito',
+      PLANNED_ABSENCE: 'Vuorotyöpoissaolo',
```
Comment (kimifoud) [inline frontend/src/lib-customizations/espoo/employee-mobile-frontend/assets/i18n/fi.ts:96]: `PLANNED_ABSENCE` now has varying labels. Should the citizen labels be updated accordingly?
image

Comment (jaahon) [reply frontend/src/lib-customizations/espoo/employee-mobile-frontend/assets/i18n/fi.ts:96]: The labels have now been updated and the option has been hidden altogether if none of the children are in shift care.

**Review by kimifoud — APPROVED** [review]

**Pull request merged**